versions.yaml

```diff
@@ -138,8 +138,8 @@ assets:
     url: "https://github.com/kata-containers/kata-containers/tools/osbuilder"
     architecture:
       aarch64:
-        name: &default-initrd-name "ubuntu"
-        version: &default-initrd-version "18.04"
+        name: &default-initrd-name "fedora"
+        version: &default-initrd-version "30"
```

Comment:
If we're going to use Fedora for that, I'd recommend we use the newest supported version (32) or even rawhide for that.

Using Fedora 30 is not exactly the best decision, as it already reached its EOL.

Reply:
It was just for my local testing. I've fixed up alpine rootfs.